crown-court-contribution/src/test/java/uk/gov/justice/laa/crime/contribution/controller/ContributionControllerContributionIntegrationTest.java

```diff
@@ -47,7 +49,10 @@ class ContributionControllerContributionIntegrationTest {
     private MockMvc mvc;
     private static final WireMockServer wiremock = new WireMockServer(9999);
     private static final String ENDPOINT_URL = "/api/internal/v1/contribution/calculate-contribution";
+    private static final String BASE_URL = "/api/internal/v1/contribution/";
```

Comment:
Is this going to cause problems? It looks like there will be a double slash.

Reply:
Fixed it